debian/changelog

```diff
@@ -0,0 +1,12 @@
+eos-data-distribution (0.1.0) UNRELEASED; urgency=medium
```

Comment:
We try to treat each codebase we build as an "upstream" project, and upstream projects usually do not have debian directories.